src/bundle/Resources/mappings/config-resolver-namespaces-map.php

```diff
@@ -37,4 +37,5 @@
     'silver_tools' => 'ibexa.commerce.site_access.config.tools',
     'siso_voucher' => 'ibexa.commerce.site_access.config.voucher',
     'st_ez_helper_create_user' => 'ibexa.commerce.site_access.config.create_user',
+    'ez_platform_automated_translation' => 'ibexa.automated_translation',
```

Comment:
This is too generic. Those settings are in the format `<namespace>.<scope>.<setting>` so if scope overlaps with any other ordinary container parameter setting, there will be a conflict.

Reply:
updated